main.js

```diff
@@ -49,41 +50,39 @@ var textInput = document.querySelector('.message-input');
 var textInputLabel = document.querySelector('.input-error');
 var errorMessage = document.querySelector('.receive-error');
 var clearMessageButton = document.querySelector('.clear-message');
-var currentMessage = '';
 
 //--------------------/event listeners/----------------------//
-// window.addEventListener('load', )
+
 receiveMessageButton.addEventListener('click', displayMessage);
 addMessageButton.addEventListener('click', displayForm);
 submitButton.addEventListener('click', submitMessage);
+clearMessageButton.addEventListener('click', showBellImage);
 addMantraSelect.addEventListener('click', hideErrorMessage);
 addAffirmationSelect.addEventListener('click', hideErrorMessage);
 mantraSelect.addEventListener('click', hideErrorMessage);
 affirmationSelect.addEventListener('click', hideErrorMessage);
-clearMessageButton.addEventListener('click', showBellImage);
 
 //--------------------/functions/----------------------//
-//transition to the data model!
-//create helper function that checks to see if newAff/newMant string exists in the aff or mant arrays.
-//create logic statement to pass it to the correct array accordingly.
 
 function submitMessage(e) {
```

Comment:
If you are passing "event" to a function, you need to wrap it in an anon function in the event listener